cvat-ui/src/reducers/notifications-reducer.ts

```diff
@@ -729,6 +732,43 @@ export default function (state = defaultState, action: AnyAction): Notifications
                 },
             };
         }
+        case ConsensusActionTypes.MERGE_CONSENSUS_JOBS_FAILED: {
+            const { taskID } = action.payload;
+            if (action.payload.error.code === 400) {
+                action.payload.error.message = "Consensus Jobs aren't annotated.";
```

Comment:
Why we specify this message on UI? Shouldnt it be coming from the server already?